### ISSUE TITLE
Selecting and copying characters such as `”` causes error in

### ISSUE THREAD

**Issue by arekkas** [ianstormtaylor/slate#161]
When selecting and copying (ctrl+c) charaters such as `”` on the demo site, I get an error on both Chrome and Firefox.

**Chrome**
```
content.js:328 Uncaught InvalidCharacterError: Failed to execute 'btoa' on 'Window': The string to be encoded contains characters outside of the Latin1 range.
```

**Firefox**
```
InvalidCharacterError: String contains an invalid character
```

This seems to be the root cause of the error:
```
var encoded = window.btoa(string);
```

**Comment by ianstormtaylor** [comment]
Fixed in `0.6.2`! Thanks for the detailed issue, made it super easy to fix!

**Comment by arekkas** [comment]
Thanks!
